### PR TITLE
[docs]: improve the design of the feedback widget.

### LINKED ISSUE #555: [docs]: improve the design of the feedback widget
image

I'll make a PR as per discussed internally with Christin in our 1:1 meeting.

### PR DESCRIPTION
**Pull request by nisarhassan12**
Fixes #555

This is how it looks now:

| Before | After |
| - | - |
| image | image 
| image | image |

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/556"></a>

### CODE REVIEW

File: src/components/docs/feedback-widget.svelte [hunk 1]
```diff
@@ -35,55 +35,62 @@
...
-              autocorrect="off"
-              type="text"
-            />
-          </div>
-          <div>
-            <span
-              ><button role="button" type="submit" class="btn-primary"
-                ><span>Send</span></button
-              ></span
+<div class="my-huge">
```
Comment (mikenikles) [inline src/components/docs/feedback-widget.svelte:38]: I don't see this class used anywhere, can we remove it?

**Review by mikenikles — APPROVED** [review]
Very well done, this looks a lot cleaner. Thank you Nisar!

File: src/components/docs/feedback-widget.svelte (hunk 1 above)
Comment (nisarhassan12) [reply src/components/docs/feedback-widget.svelte:38]: Thanks. I added it for 2 reasons one I wanted spacing huge to be applied across the y-direction I could have added it to the white box itself but the main usage of this is being able to use `mx-auto` on the white box to centre it.

Comment (mikenikles) [reply src/components/docs/feedback-widget.svelte:38]: Oh my 🤣... I thought this was a custom CSS class name called `my-huge` as in some kind of test name. Reading your comment, I now realize this is "margin", "y" with a custom size "huge".

If I ever need a good reason to showcase why using Tailwind's default, digit-based scale is better than a name-based one, this right here is it 😄.

**Pull request merged**